contracts/root/withdrawManager/WithdrawManager.sol

```diff
@@ -237,8 +305,8 @@ contract WithdrawManager is WithdrawManagerStorage /* , IWithdrawManager */ {
     uint256 _exitId = (
       headerNumber * HEADER_BLOCK_NUMBER_WEIGHT +
       withdrawBlockNumber * WITHDRAW_BLOCK_NUMBER_WEIGHT +
-      path.toRlpItem().toBytes().toRlpItem().toUint() * 100000 +
-      oIndex
+      path.toRlpItem().toBytes().toRlpItem().toUint() * 100000
+      // + oIndex
```

Comment:
Will exit Ids be the same?

Reply:
Yes, the way we calculate the *age* will remain same like in MVL. Just that we will take the age of reference tx now. I'll probably give an exit ID that'll be different from *age*, like sequential maybe - Yet to think about the *implementation details* of the challenges, so using age right now.